src/ignitus-Shared/Components/InternshipList/style.scss

```diff
@@ -0,0 +1,54 @@
+.internship-list-item {
```

Comment:
It doesn't make sense to write a component a combination of `scss` and `emotion`, please go with with of the one approach, I would say go with `emotion`.